build.gradle

```diff
@@ -62,7 +62,7 @@ allprojects {
   }
 
   dependencies {
-    compile group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '6.1'
+    compile group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '6.2'
```

Comment:
hmm, feeling like this might be an unused dependency

Reply:
i think your fear/feel is correct - don't see `JSON_CONSOLE` in this repo anymore